w1r3/README.md

```diff
@@ -0,0 +1,127 @@
+# Developing and Deploying the W1R3 benchmarks
+
+This document describes how to deploy and develop the W1R3 benchmarks. The
+motivation and objectives of these benchmarks are out of scope, please see the
+design documents for details.
+
+## Running locally
+
+The benchmarks can be run locally, from the command line. In all these examples
+we will assume you have set the `PROJECT_ID` shell variable to a project you
+use for development, and `BUCKET_NAME` to the name of a bucket enabled for gRPC.
+
+Please do not use the "production" project, we want to minimize the chances of
+losing historical data, or adding noise to it.
+
+The metrics will be published to `PROJECT_ID`. You will use the
+[Metrics Explorer](https://console.google.com/monitoring/metrics-explorer) to
+see the metric results.
+
+Note that the `GRPC+DP` transport is disabled, as it is unlikely to work from
+your workstation.
+
+- C++
+  ```shell
+  cmake --build .build/ && .build/w1r3/cpp/w1r3 \
+    --project-id=${PROJECT_ID} \
+    --bucket=${BUCKET_NAME} \
+    --transport=GRPC+CFE --transport=JSON \
+    --iterations=10000
+  ```
+- Go
+  ```shell
+  go -C w1r3/go build && ./w1r3/go/w1r3 \
+    -project-id=${PROJECT_ID} \
+    -bucket=${BUCKET_NAME} \
+    -transport GRPC+CFE -transport JSON \
+    -iterations 10000
+  ```
+- Java
+  ```shell
+  env -C w1r3/java mvn dependency:resolve-plugins && \
```

Comment:
For dev, we can remote this explicit step, as it will be performed as part of `mvn package` and we're not trying to optimize and docker layers.

Reply:
Done.